roles/comanage-registry/defaults/main.yml

```diff
@@ -7,7 +7,8 @@ comanage_org_identity_pooling: "No"
 #  family_name: "Jones Smith"
 #  username: "pat.jones.smith@university.edu"
 
-comanage_version: "rciam-3.1.x"
+# Override comanage_version with a tag name. This will result to a new directory name comanage-registry-<tag>
```

Comment:
COmanaged will be checkout out under `{{ comanage_root_dir }}/comanage-registry-{{ comanage_version }}`

Reply:
Done